test/integration/test_slow.py

```diff
@@ -127,7 +127,10 @@ def test_install_role_requirements():
     env = ScriptTestEnvironment()
     result = env.run('ansible-container', '--debug', 'build',
                      cwd=project_dir('requirements'), expect_stderr=True)
-    assert "ansible-role-apache was installed successfully" in result.stderr
+    print result.stdout
+    print "YYYYYYYYYYYYYYYYYYYYYYYYYYY\n"
```

Comment:
We're going from a known working versions of docker-compose, docker-py, Jinja2 and PyYaml to just installing the latest. It seems that installing the latest of one or more of these packages broke something, because this is the only place where this test fails.

Can you please dig a little deeper and figure out what's actually going on. I don't think inserting a hack just to make the test work should be an acceptable solution. It seems to me that we run the risk of having this bug surface in places where the hack is not implemented.